testing/src/main/java/org/oppia/testing/TestLogReportingModule.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.testing
+
+import dagger.Module
+import dagger.Provides
+import java.lang.Exception
+import javax.inject.Singleton
+import org.oppia.util.firebase.CrashLogger
+
+/**
+ * Provides fake crash logging dependencies.
+ */
+@Module
+class TestLogReportingModule {
+  companion object {
+    var exceptionList = ArrayList<Exception>()
+  }
+
+  @Provides
+  @Singleton
```

Comment:
If you make the fake crash logger an injectable ``@Singleton``, you don't need to redo that here since calling the provider at each injection site for ``CrashLogger`` will reuse the same fake crash logger. Instead, you can bind it to keep it simple:

```kotlin
@Module
interface TestLogReportingModule {
  @Binds
  fun bindFakeCrashLogger(FakeCrashLogger impl): CrashLogger
}
```